server.go

```diff
@@ -86,29 +99,42 @@ func (s *StubServer) HandleRequest(w http.ResponseWriter, r *http.Request) {
 	start := time.Now()
 	fmt.Printf("Request: %v %v\n", r.Method, r.URL.Path)
 
+	// Prepare internal error as it's the most commonly used here
+	stripeError := &ResponseError{
```

Comment:
Instead of preparing this as a variable, can we instead have a function that prepares an error response and returns it? I think that would be slightly clearer.

Reply:
@tmaxwell-stripe fixed in https://github.com/stripe/stripe-mock/pull/32/commits/f9ad1e76482ce44b1eb9367cc8485dac93dd39a1 is this what you had in mind?

Reply:
Yep, that's it. Thanks!